app/assets/stylesheets/spree/frontend/components/layout/_header.scss

```diff
@@ -16,7 +16,46 @@
     &__content {
       align-items: center;
       display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
       padding: 2rem 3rem;
+
+      @include desktop {
+        flex-wrap: nowrap;
+        justify-content: flex-end;
+      }
+
+      .logo {
+        order: 1;
+
+        img {
+          width: 10em;
+
+          @include tablet {
+            width: auto;
+          }
+        }
+      }
+
+      .search-bar {
+        flex: 0 0 100%;
+        margin-top: 10px;
+        order: 4;
+
+        @include desktop {
+          flex: 1 0 auto;
+          margin-top: 0;
+          order: 2;
+        }
+      }
+
+      .auth-link {
+        order: 2;
+      }
+
+      .cart-link {
```

Comment:
We should move this styles to `_cart_link.scss`. Is there a reason why they are here?